apps-rendering/src/components/shared/card.tsx

```diff
@@ -44,51 +44,79 @@ interface Props {
 	image: Option<Image>;
 }
 
-const borderColor = (
-	type: RelatedItemType,
-	format: ArticleFormat,
-): SerializedStyles => {
-	if (type === RelatedItemType.ADVERTISEMENT_FEATURE) {
-		return css`1px solid ${labs[300]}`;
-	} else {
-		return css`1px solid ${getThemeStyles(format.theme).kicker}`;
-	}
-};
-
 const listStyles = (
 	type: RelatedItemType,
 	format: ArticleFormat,
 ): SerializedStyles => {
-	return css`
-		background: white;
-		margin-right: ${remSpace[2]};
-		flex: 0 0 42vw;
-		display: flex;
-		flex-direction: column;
-		justify-content: space-between;
-		border-top: ${borderColor(type, format)};
-		max-width: 10rem;
-
-		&.fade {
-			opacity: 0.7;
-		}
+	switch (type) {
+		case RelatedItemType.VIDEO:
+		case RelatedItemType.AUDIO:
+		case RelatedItemType.GALLERY: {
+			return css`
+				margin-right: ${remSpace[2]};
+				flex: 0 0 42vw;
+				display: flex;
+				flex-direction: column;
+				justify-content: space-between;
+				border-radius: ${remSpace[2]};
+				max-width: 10rem;
+				padding-top: 0.125rem;
+
+				&.fade {
+					opacity: 0.7;
+				}
 
-		${darkModeCss`
-            background: ${neutral[7]};
-        `}
+				${darkModeCss`
+					background: ${neutral[10]};
+				`}
 
-		${from.tablet} {
-			margin-right: ${remSpace[5]};
-		}
+				${from.tablet} {
+					margin-right: ${remSpace[5]};
+				}
 
-		${from.desktop} {
-			max-width: 13.75rem;
+				${from.desktop} {
+					max-width: 13.75rem;
+				}
+
+				&:last-of-type {
+					margin-right: 0;
+				}
+			`;
 		}
+		default: {
+			return css`
+				margin-right: ${remSpace[2]};
+				flex: 0 0 42vw;
+				display: flex;
+				flex-direction: column;
+				justify-content: space-between;
+				border-top: 1px solid ${neutral[86]};
```

Comment:
Is it worth trying to only update the fields that need to be updated instead of a switch statement in order to avoid duplicated CSS

```


const isDefaultStyle = type !==  RelatedItemType.VIDEO && type !== RelatedItemType.AUDIO && type !== RelatedItemType.GALLERY;

return css`
    margin-right: ${remSpace[2]};
    flex: 0 0 42vw;
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    ${isDefaultStyle ? `border-top: 1px solid 1px solid ${neutral[86]};` : ''}
    ${isDefaultStyle ? `0 0 ${remSpace[2]} ${remSpace[2]}` : ''}
    border-radius: ${remSpace[2]};
    max-width: 10rem;
    padding-top: 0.125rem;
    &.fade {
        opacity: 0.7;
    }
    ${darkModeCss`
        ${borderTop}
        background: ${neutral[10]};
    `}
    ${from.tablet} {
        margin-right: ${remSpace[5]};
    }
    ${from.desktop} {
        max-width: 13.75rem;
    }
    &:last-of-type {
        margin-right: 0;
    }
`
```